### PR TITLE
Use djmail and templates for mail sending

### PR DESCRIPTION
**Pull request by catherinedevlin**
Does not change existing password reset emails directly - that is provided by django-rest-auth.

Other emails can now be sent with instructions included as comment in views.py.

### CODE REVIEW

File: afbirez/settings/base.py [hunk 1]
```diff
@@ -95,3 +95,5 @@
 )
 
 DEFAULT_FROM_EMAIL='catherine.devlin@gsa.gov'
```
Comment (DavidEBest) [inline afbirez/settings/base.py]: @catherinedevlin Could we replace this with a dummy address, something like sbirez@gsa.gov? Would those emails be sent?

**Comment by DavidEBest** [comment]
Is this still WIP?

**Comment by catherinedevlin** [comment]
Finally got this confirmed.  Ready for merge AFAIK.

Still sends from catherine.devlin@gsa.gov, but let's make a separate PR for that later, since I don't know how long it will take to get that address fully enabled with Amazon SES.

**Comment by DavidEBest** [comment]
Looks like the other two PRs from today have made this unmergeable. Could you re-merge master?

**Comment by catherinedevlin** [comment]
Will do...

**Comment by DavidEBest** [comment]
Test complains about a missing template: `emails/submit_notification-body-html.html`. Does that file need checked in?

**Comment by catherinedevlin** [comment]
Apparently djmail checks for such a file and uses it if it's present, but it's not actually needed here.  If we wanted an HTML version of the mail, we'd write that.

I don't know offhand how to keep it from looking like an error in the test results.  :(

**Comment by DavidEBest** [comment]
That's fine, then. Just wanted to make sure it wasn't a forgotten file.

**Pull request merged**